ndcube/tests/test_ndcube.py

```diff
@@ -230,6 +230,13 @@ def test_axis_world_coords_wave_ec(ndcube_3d_l_ln_lt_ectime):
     assert coords[0].shape == (5,)
 
 
+@pytest.mark.limit_memory("12 MB")
```

Comment:
I grafted this test into the commit history before @nabobalis original #780 and it fails (using 3GB of RAM). 

This test doesn't actually fail before this PR though, as this change fixes the case where you want to extract a single world coordinate correlated with two pixel axes where those pixel axes are also correlated with other world axes, i.e. the VISP WCS.

Reply:
I have now added a test which fails like this DKIST WCS did, but without the need to actually load the DKIST data.